### ISSUE TITLE
Synchronized Script has DROP SCHEMA's

### ISSUE THREAD

**Issue by TonyDeBuys** [OpenDBDiff/OpenDBDiff#115]
OpenDBDiff: OpenDBDiff_0.10.0

When comparing two schemas OpenDBDiff identified the differences correctly, but the generated Synchronized script has several DROP statements at the then, including DROP SCHEMA which drops the schema where my tables are...

The tail end of the script looks like this:

```
CREATE procedure [idi].[*]
as
begin
    *
end
GO
DROP SCHEMA [idi_tmp]
GO
DROP SCHEMA [rnd]
GO
DROP SCHEMA [gdi_atomic]
GO
DROP SCHEMA [idi]
```

(For brevity I've removed the procedure name and body)
But, in this case, idi (and the other schema's that are dropped) are schema's that I need!

**Comment by jspraul** [comment]
This sounds like a bad bug! I tried comparing the schemas in the attached simple test databases and got the results I expected, without the script dropping the schemas at the end.

Have you made any changes in the comparison options?

[20190811-opendbdiff-schema-test.txt](https://github.com/OpenDBDiff/OpenDBDiff/files/3489891/20190811-opendbdiff-schema-test.txt)

**Comment by igitur** [comment]
No, no major changes to comparison options recently. I'll check this out soon. Will see if I can find a recent commit that might have introduced this. Again, the needs for a proper unit test framework comes to mind ;-)

**Comment by igitur** [comment]
Oh, I thought @jspraul was mentioning me. 

I can't reproduce this. 

@TonyDeBuys Please fill out the complete [issue template](https://github.com/OpenDBDiff/OpenDBDiff/blob/develop/.github/ISSUE_TEMPLATE.md) so that we may have all the required information.
